nw-firestore-schema.ts

```diff
@@ -81,6 +81,7 @@ interface Application {
     gender: string;
     ethnicity: string[];
     isOfLegalAge: boolean;
+    phoneNumber: number;
```

Comment:
This should probably be a string
```suggestion
    phoneNumber: string;
```